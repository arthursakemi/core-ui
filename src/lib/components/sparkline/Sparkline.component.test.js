import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/sparkline/__snapshots__"
});
