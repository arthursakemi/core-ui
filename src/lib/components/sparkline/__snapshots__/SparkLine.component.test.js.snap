// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots SparkLine Default 1`] = `
<div
  className="sc-bwzfXH hilSGa"
>
  <h3
    className="sc-htpNat bnCVRm"
  >
    Vege-Lite sparkline chart demo
  </h3>
  <div
    className="sc-bdVaJa bDWFJH sc-vegachart"
    id="vis"
  />
  <h3
    className="sc-htpNat bnCVRm"
  >
    Vege-Lite sparkline multi chart demo
  </h3>
  <div
    className="sc-bdVaJa bDWFJH sc-vegachart"
    id="multiVis"
  />
</div>
`;
